### PR TITLE
Added GitHub actions CI

### PR DESCRIPTION
**Pull request by matusvalo**
This PR Adds 1:1 replacement of Travis CI  with GitHub actions.

### CODE REVIEW

**Comment by matusvalo** [comment]
Next step is kombu...

**Comment by auvipy** [comment]
> Next step is kombu...

you can improve https://github.com/celery/kombu/pull/1299

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        uses: actions/checkout@v2.3.4
+      - name: Set up Python ${{ matrix.python-version }}
+        id: python
+        uses: actions/setup-python@v2.2.1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: pip install --upgrade pip setuptools wheel tox tox-docker
+      - name: Run unittest
+        run: tox -v -e ${{ matrix.python-version }}-unit -- -v
```
Comment (auvipy) [inline .github/workflows/ci.yaml:44]: when I first tried using tox integration tests in celery took so long time that I had to try other options. good to see it working good

Comment (matusvalo) [reply .github/workflows/ci.yaml:44]: Yes GitHub actions are pretty fast.

**Comment by matusvalo** [comment]
Can we merge even this PR? Are there any objections @auvipy @thedrow ?

**Review by auvipy — APPROVED** [review]
we can improve as we go.

**Pull request merged**